nixos/modules/config/networking.nix

```diff
@@ -223,7 +223,9 @@ in
             '';
 
       } // optionalAttrs config.services.resolved.enable {
-        "resolv.conf".source = "/run/systemd/resolve/resolv.conf";
+        # symlink the static version of resolv.conf as recommended by upstream:
+        # https://www.freedesktop.org/software/systemd/man/systemd-resolved.html#/etc/resolv.conf
+        "resolv.conf".source = "${pkgs.systemd}/lib/systemd/resolv.conf";
```

Comment:
I fear this breaks some people's setup using `openresolv` and `dhcpcd`  - and enforce the use of `networkd`. I think resolved will longer be able dynamic dns servers this way (note that the service behaves differently depending on the existence of this symlink).